shared/components/NextCommunityEvent/index.js

```diff
@@ -12,9 +13,19 @@ const calendarURL = 'https://calendar.google.com/calendar/event?action=TEMPLATE'
   '&tmeid=NWY0cDE3Y3N0MzZhbWp2amxmdjhkdHBqbGsgbG5kaDVzdXRrbmtyZjZpbjEzYWgzYmUwbW9AZw' +
   '&tmsrc=lndh5sutknkrf6in13ah3be0mo%40group.calendar.google.com';
 
-const locationURL = 'https://goo.gl/maps/Z8SU87i4Fy42';
+function googleMapsUrl(location) {
+  const { latitude, longitude } = pathOr({}, ['coordinates'], location);
+  if (!latitude && !longitude) { return null; }
```

Comment:
Or instead of and? If either are undefined we don't want to generate a gmap link.